### PR TITLE
Merge back 2.2.0.rc3 to develop

### PR DESCRIPTION
**Pull request by kprokofi**
### Checklist

- [ ] I have added unit tests to cover my changes.​
- [ ] I have added integration tests to cover my changes.​
- [ ] I have ran e2e tests and there is no issues.
- [ ] I have added the description of my changes into CHANGELOG in my target branch (e.g., [CHANGELOG](https://github.com/openvinotoolkit/training_extensions/blob/develop/CHANGELOG.md) in develop).​
- [ ] I have updated the documentation in my target branch accordingly (e.g., [documentation](https://github.com/openvinotoolkit/training_extensions/tree/develop/docs) in develop).
- [ ] I have [linked related issues](https://help.github.com/en/github/managing-your-work-on-github/linking-a-pull-request-to-an-issue#linking-a-pull-request-to-an-issue-using-a-keyword).

### License

- [ ] I submit _my code changes_ under the same [Apache License](https://github.com/openvinotoolkit/training_extensions/blob/develop/LICENSE) that covers the project.
      Feel free to contact the maintainers if that's a concern.
- [ ] I have updated the license header for each file (see an example below).

```python
# Copyright (C) 2024 Intel Corporation
# SPDX-License-Identifier: Apache-2.0
```

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -45,15 +45,25 @@ All notable changes to this project will be documented in this file.
   (<https://github.com/openvinotoolkit/training_extensions/pull/3769>)
 - Refactoring `ConvModule` by removing `conv_cfg`, `norm_cfg`, and `act_cfg`
   (<https://github.com/openvinotoolkit/training_extensions/pull/3783>, <https://github.com/openvinotoolkit/training_extensions/pull/3816>, <https://github.com/openvinotoolkit/training_extensions/pull/3809>)
+- Support ImageFromBytes
+  (<https://github.com/openvinotoolkit/training_extensions/pull/3948>)
+- enable model export
```
Comment (sungchul1) [inline CHANGELOG.md]:
```suggestion
- Enable model export
```
To use a capital letter

File: src/otx/recipe/detection/atss_resnext101_tile.yaml [hunk 2]
```diff
@@ -0,0 +1,51 @@
+model:
+  class_path: otx.algo.detection.atss.ResNeXt101ATSS
```
Comment (harimkang) [inline src/otx/recipe/detection/atss_resnext101_tile.yaml]: this is where the refactoring of develop is not reflected. (all tiling recipes)

Comment (kprokofi) [reply src/otx/recipe/detection/atss_resnext101_tile.yaml]: OK, will check today

**Comment by harimkang** [comment]
@eugene123tw Hi Eugene, Something is conflicting between 2.2.0.rc2 and develop with tiling, getting unit-test fails in s-seg tiling case (tests/unit/core/data/test_tiling.py::TestOTXTiling::test_tile_transform[SEMANTIC_SEGMENTATION]).
image

Can you take a look?

**Comment by eugene123tw** [comment]
@kprokofi Could you add fixes in this PR? https://github.com/openvinotoolkit/training_extensions/pull/3973/files

**Review by harimkang — APPROVED** [review]

**Review by sovrasov — APPROVED** [review]

**Comment by sovrasov** [comment]
@kprokofi could you provide a result of a local tests launch? CI doesn't work today

**Comment by kprokofi** [comment]
detection tests are passed locally:
image

**Pull request merged**